build-parent/pom.xml

```diff
@@ -81,13 +81,13 @@
         <junit-pioneer.version>2.2.0</junit-pioneer.version>
 
         <!-- Database images for JDBC/Reactive/Hibernate tests and devservices -->
-        <postgres.image>docker.io/postgres:14</postgres.image>
+        <postgres.image>docker.io/postgres:16</postgres.image>
         <mariadb.image>docker.io/mariadb:10.11</mariadb.image>
         <db2.image>icr.io/db2_community/db2:11.5.9.0</db2.image>
         <mssql.image>mcr.microsoft.com/mssql/server:2022-latest</mssql.image>
-        <mysql.image>docker.io/mysql:8.0</mysql.image>
+        <mysql.image>docker.io/mysql:8.4</mysql.image>
         <oracle.image>docker.io/gvenzl/oracle-free:23-slim-faststart</oracle.image>
-        <mongo.image>docker.io/mongo:4.4</mongo.image>
+        <mongo.image>docker.io/mongo:7.0.12</mongo.image>
```

Comment:
I think that 7.0 should be used as it seems to point to latest 7.0.x.

Tested it with `docker run --name some-mongo -d mongo:7.0` and see that 7.0.12 version is used.

Reply:
Push changed to 7.0